src/_playground/documentation/utils/SetMeta.js

```diff
@@ -0,0 +1,19 @@
+import Helmet from 'react-helmet-async';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const SetMeta = ({ title, description }) => {
+    return (
+        <Helmet>
+            <title>{title}</title>
+            <meta content={description} name='description' />
+        </Helmet>
+    );
+};
+
+SetMeta.propTypes = {
+    description: PropTypes.string.isRequired,
```

Comment:
Should `description` be required?  Seems like it could be optional.  The `description` prop in `ComponentPage` and `MarkdownPage` is optional so there might be work to do in those components if this needs to remain required.